platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/ClaimsHandlerManager.java

```diff
@@ -230,6 +232,19 @@ protected LDAPConnectionFactory createLdapConnectionFactory(String url, Boolean
         } catch (NumberFormatException ignore) {
         }
 
+        try {
```

Comment:
Same comment about pulling this block out to a private method.

In this case, it's even a little worse because it's an exact dupe of the auditing block in `SslLdapLoginModule`.